source/tutorial/configure-x509-member-authentication.txt

```diff
@@ -82,9 +82,9 @@ Use Command-line Options (``tls``)
 
 .. note::
 
-   The procedures in this section use the ``ssl`` settings/option. For
-   procedures using their ``tls`` aliases (Available in MongoDB 4.2),
-   see :ref:`configure-member-tls`.
+   The procedures in this section use the ``tls`` settings/option. For
```

Comment:
Can we use the existing copy from https://github.com/mongodb/docs/blob/master/source/tutorial/configure-x509-client-authentication.txt#L66-L74 ?

I'm hesitant to write in 'all versions' as we may remove SSL in the future (and have to go back and remember to fix this at that point in time). 

Alternatively, tweak to just state 'For procedures using the deprecated ``ssl`` aliases, see <blah>'.

Reply:
Changed to the last paragraph, existing wording doesn't make sense (ssl options are available before 4.2).